libcontainer/cgroups/fs2/devices.go

```diff
@@ -3,35 +3,27 @@
 package fs2
 
 import (
+	"strings"
+
 	"github.com/opencontainers/runc/libcontainer/cgroups/ebpf"
 	"github.com/opencontainers/runc/libcontainer/cgroups/ebpf/devicefilter"
 	"github.com/opencontainers/runc/libcontainer/configs"
 	"github.com/pkg/errors"
 	"golang.org/x/sys/unix"
 )
 
-func isRWM(cgroupPermissions string) bool {
-	r := false
-	w := false
-	m := false
-	for _, rn := range cgroupPermissions {
-		switch rn {
-		case 'r':
-			r = true
-		case 'w':
-			w = true
-		case 'm':
-			m = true
-		}
-	}
+func isRWM(perm string) bool {
+	r := strings.Contains(perm, "r")
+	w := strings.Contains(perm, "w")
+	m := strings.Contains(perm, "m")
```

Comment:
Looks like the old implementation is about 3x more efficient. Not that this is a critical part :smile: but I don't see a reason to replace it.

Reply:
I would do that for readability but to my mind both the old and the new one is fine.

Reply:
Yeah, I'm not actually sure why I did this tbh. :thinking: